modules/apps/translation/translation-translator-aws/src/main/java/com/liferay/translation/translator/aws/internal/translator/AWSTranslator.java

```diff
@@ -101,15 +102,25 @@ protected void activate(Map<String, Object> properties) {
 		_awsTranslatorConfiguration = ConfigurableUtil.createConfigurable(
 			AWSTranslatorConfiguration.class, properties);
 
-		_translateClient = TranslateClient.builder(
-		).credentialsProvider(
-			StaticCredentialsProvider.create(
-				AwsBasicCredentials.create(
-					_awsTranslatorConfiguration.accessKey(),
-					_awsTranslatorConfiguration.secretKey()))
-		).region(
-			Region.of(_awsTranslatorConfiguration.region())
-		).build();
+		if (_awsTranslatorConfiguration.enabled()) {
+			_translateClient = TranslateClient.builder(
+			).credentialsProvider(
+				StaticCredentialsProvider.create(
+					AwsBasicCredentials.create(
+						_awsTranslatorConfiguration.accessKey(),
+						_awsTranslatorConfiguration.secretKey()))
+			).region(
+				Region.of(_awsTranslatorConfiguration.region())
+			).build();
+		}
+		else {
+			_translateClient = null;
```

Comment:
We are assigning `_translateClient` to null value. 
There is some way that can we try to use it being null? (on the line 129) (like being enabled and after disabling) or it's a very extreme corner case.
it's worth a NPE check?